.travis.yml

```diff
@@ -136,8 +139,11 @@ before_install:
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then rm /usr/local/include/c++ ; fi
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew ls --versions gcc    > /dev/null || brew install gcc;    fi
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew ls --versions gnupg2 > /dev/null || brew install gnupg2; fi
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew ls --versions ccache > /dev/null || brew install ccache; fi
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then virtualenv venv; fi
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then source venv/bin/activate; fi
+  - ccache -M 2G && ccache -z
```

Comment:
Should this move after the next line on osx (otherwise, how is ccache found there?)

Reply:
No, the line below it to add the compiler wrappers to the path, `ccache` is already in the path.